lib/Runtime/Library/SharedArrayBuffer.cpp

```diff
@@ -452,12 +452,9 @@ namespace Js
             {
                 HeapDeleteArray(sharedContents->bufferLength, sharedContents->buffer);
             }
-
-            if (!isShutdown)
-            {
-                Recycler* recycler = GetType()->GetLibrary()->GetRecycler();
-                recycler->ReportExternalMemoryFree(sharedContents->bufferLength);
-            }
+                        
```

Comment:
@jianchun maybe I remember wrong but.. do you think this has anything to do with exit time cleanup we were dealing?